src/mixins/action-mixin.ts

```diff
@@ -0,0 +1,116 @@
+import { Component, Mixins } from 'vue-property-decorator'
+import { Action, Getter } from 'vuex-class'
+import querystring from 'qs'
+import axios from 'axios'
+import { Debounce } from 'vue-debounce-decorator'
+import canadaPostAPIKey from '@/store/config'
+import { ActionBindingIF } from '@/interfaces/store-interfaces'
+import { ApplicantI } from '@/interfaces'
+import { ApiErrorMixin } from '@/mixins'
+
+@Component
+export class ActionMixin extends Mixins(ApiErrorMixin) {
```

Comment:
New mixin with stuff I pulled out of `store/actions.ts` (hence the name).

Do what you want with this. But we need somewhere for these methods to live where they have `this` in scope (so, not the store actions).